src/features/device-logs/lib/backends/redis.ts

```diff
@@ -28,6 +28,7 @@ const UNSUBSCRIBECMD = REDIS_LOGS_SHARDED_PUBSUB
 	? 'sunsubscribe'
 	: 'unsubscribe';
 const PUBLISHCMD = REDIS_LOGS_SHARDED_PUBSUB ? 'spublish' : 'publish';
+const MESSAGECMD = REDIS_LOGS_SHARDED_PUBSUB ? 'smessage' : 'message';
```

Comment:
Where did you find that :sparkles: ?
The only place I could find is mentioned but w/o any docs was v
See: https://redis.io/commands/ssubscribe/

Reply:
I verified it there but I found it based upon the ioredis PR that added support (it added support for emitting an `smessage` command which raised the question)